src/Iteration.cpp

```diff
@@ -97,6 +97,32 @@ Iteration::setTimeUnitSI(double newTimeUnitSI)
     return *this;
 }
 
+Iteration &
+Iteration::close( bool flush )
+{
+    setAttribute( "closed", true );
```

Comment:
do we need an attribute or would a private member be sufficient?

Reply:
> e.g. for streaming reasons

That's exactly the reason, yes. I use this in the data sink to know which iterations I need not parse any more.

Reply:
Sounds good then :)